### PR TITLE
Bugfix/hportal

### PR DESCRIPTION
**Pull request by tacketar**

### CODE REVIEW

File: src/gop/hportal.c [hunk 1]
```diff
@@ -183,9 +183,8 @@ void _reap_hportal(gop_host_portal_t *hp, int quick)
                 destroy_host_connection(hc);
 
                 break;
-            } else {  //** Got somone trying ot close it so wait a little bit
+            } else {  //** Got somone trying to close it so just return
```
Comment (PerilousApricot) [inline src/gop/hportal.c]: Lol, you corrected one typo, but left 'somone'

File: src/lio/ex3.c [hunk 2]
```diff
@@ -337,11 +337,17 @@ ex_id_t exnode_exchange_get_default_view_id(lio_exnode_exchange_t *exp)
 lio_exnode_exchange_t *lio_lio_exnode_exchange_text_parse(char *text)
 {
     lio_exnode_exchange_t *exp;
+    tbx_inip_file_t *ifd;
 
-    exp = lio_exnode_exchange_create(EX_TEXT);
+    ifd = tbx_inip_string_read(text);
+    if (ifd == NULL) {
+        log_printf(0, "ERROR: tbx_inip_string_read() returned NULL!\n"); tbx_log_flush();
+	return(NULL);
```
Comment (PerilousApricot) [inline src/lio/ex3.c]: Indent

**Comment by tacketar** [comment]
Nitpicker!  At least the comment accurately describes what it's doing:)  
It's been changed and pushed.

**Comment by tacketar** [comment]
It's been changed.  The box I made the changes on wasn't my normal box 
and it had a tab vs spaces.

**Comment by PerilousApricot** [comment]
This is OK. I'll cherry-pick to get rid of the merge bubble once the other PRs are done, so I don't have to rebase 4 patches

**Pull request closed without merging**